backend/django/core/views/api_annotate.py

```diff
@@ -436,9 +437,14 @@ def data_unlabeled_table(request, project_pk):
     data = []
     for d in unlabeled_data:
         serialized_data = DataSerializer(d, many=False).data
+
```

Comment:
I get the following error when clicking on the "annotate" page in any of the three sample projects 

```
smart_backend    | Internal Server Error: /api/data_unlabeled_table/3/
smart_backend    | Traceback (most recent call last):
smart_backend    |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
smart_backend    |     response = get_response(request)
smart_backend    |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
smart_backend    |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
smart_backend    |   File "/usr/local/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
smart_backend    |     return view_func(*args, **kwargs)
smart_backend    |   File "/usr/local/lib/python3.8/site-packages/django/views/generic/base.py", line 70, in view
smart_backend    |     return self.dispatch(request, *args, **kwargs)
smart_backend    |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
smart_backend    |     response = self.handle_exception(exc)
smart_backend    |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
smart_backend    |     self.raise_uncaught_exception(exc)
smart_backend    |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
smart_backend    |     raise exc
smart_backend    |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
smart_backend    |     response = handler(request, *args, **kwargs)
smart_backend    |   File "/usr/local/lib/python3.8/site-packages/rest_framework/decorators.py", line 50, in handler
smart_backend    |     return func(*args, **kwargs)
smart_backend    |   File "/code/core/views/api_annotate.py", line 443, in data_unlabeled_table
smart_backend    |     sorted_similarity_pairs.append(json.loads(pair))
smart_backend    |   File "/usr/local/lib/python3.8/json/__init__.py", line 357, in loads
smart_backend    |     return _default_decoder.decode(s)
smart_backend    |   File "/usr/local/lib/python3.8/json/decoder.py", line 337, in decode
smart_backend    |     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
smart_backend    |   File "/usr/local/lib/python3.8/json/decoder.py", line 355, in raw_decode
smart_backend    |     raise JSONDecodeError("Expecting value", s, err.value) from None
smart_backend    | json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```